### PR TITLE
Scripts fix

### PR DESCRIPTION
**Pull request by danbider**
please review and merge these changes to develop and then main.
we can talk later about using fiftyone more interactively, and about using persistent datasets that can be read into an ipython session.
tests are passing and everything should run properly for a new dataset. modified the readme too -- can still be improved, feel free to modify it further as you go tomorrow. I still need to modify toy dataset to have it with the same format as all our other datasets.

### CODE REVIEW

File: scripts/configs/eval/eval_params.yaml
Comment (themattinthehatt) [inline scripts/configs/eval/eval_params.yaml:21]: would be helpful to update the comment for each parameter in this file, users should ideally be able to read this file and understand exactly what each of these is for

File: lightning_pose/utils/scripts.py
Comment (themattinthehatt) [inline lightning_pose/utils/scripts.py:246]: :)

File: lightning_pose/utils/plotting_utils.py
Comment (themattinthehatt) [inline lightning_pose/utils/plotting_utils.py:205]: any reason to keep this function around?

File: lightning_pose/utils/plotting_utils.py
Comment (themattinthehatt) [inline lightning_pose/utils/plotting_utils.py:124]: i keep forgetting what the differences are between all these predict functions; could add a simple docstring like """Predict poses on a single video from a config and model checkpoint file."""

File: lightning_pose/utils/plotting_utils.py
Comment (themattinthehatt) [inline lightning_pose/utils/plotting_utils.py:366]: """Predict poses on a single video from an already initialized model and data loader."""

File: lightning_pose/utils/fiftyone.py
Comment (themattinthehatt) [inline lightning_pose/utils/fiftyone.py:1]: small ocd comment: to keep imports organized i usually do the following: import outside functions/modules first, in alphabetical order, skip a line, then import package-level functions/modules next, also in alphabetical order:

```python
import fiftyone as fo
import numpy as np
from omegaconf import DictConfig, OmegaConf, ListConfig
import os
import pandas as pd
from tqdm import tqdm
from typeguard import typechecked
from typing import Dict, List, Optional, Union, Callable

from lightning_pose.utils.io import return_absolute_path, return_absolute_data_paths
from lightning_pose.utils.plotting_utils import get_videos_in_dir

```

not recommending we do this for every file right now, but would be good to keep this in mind when importing new things to slow the increase of entropy

**Review by themattinthehatt — APPROVED** [review]
Looks great! I left some comments, they don't necessarily need to be addressed before merging as they're all minor/stylistic. I'll approve merging, and let you update if you'd like, otherwise feel free to merge yourself.

File: scripts/predict_new_vids.py
Comment (danbider) [reply scripts/predict_new_vids.py:23]: moved to `lightning_pose.utils.io` with all the other path handling funcs.

File: lightning_pose/utils/plotting_utils.py
Comment (danbider) [reply lightning_pose/utils/plotting_utils.py:124]: I added detailed docstrings. If you have 10-15 minutes, it'll be good if you can to take a look at these funcs yourself too. there are some redundant arguments there `data_name`, `save_preds`. If you can help me polish it, it'll help.

File: lightning_pose/utils/fiftyone.py
Comment (danbider) [reply lightning_pose/utils/fiftyone.py:1]: ok -- that makes sense. wasn't aware of this method; will start doing that from now on.

**Comment by danbider** [comment]
hey -- did all changes. thanks for taking a look.
It'll be useful if there's another pair of eyes on the `plotting_utils.py` funcs. should take just a few minutes to clean up and refine. then you could merge? going to Ram.

**Review by themattinthehatt — APPROVED** [review]

**Pull request merged**
